src/Components/DrumLoopTile/index.test.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import LoopTile from "./index";
+
+import Enzyme, { shallow, render, mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import Adapter from "enzyme-adapter-react-16";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+/*it("renders correctly enzyme", () => {
```

Comment:
Is it ok if we just remove these commented code? :)

Reply:
There's no problem, It has been done